yroots/IntervalChecks.py

```diff
@@ -937,7 +937,9 @@ def quadratic_check_nd(test_coeff, intervals, tol):
             if len(where_nonzero) == 2:
                 #coeff of cross terms
                 i,j = where_nonzero
-                #with symmetric matrices, we only need the upper part
+                #with symmetric matrices, we only need to store the lower part
+                A[j,i] = test_coeff[spot].copy()
+                #todo: see if we can store this in only one half of A
                 A[i,j] = test_coeff[spot].copy()
```

Comment:
We may be able to make this (slightly) faster by not calling np.copy() again while making it a symmetric matrix.
That is,
`A[j,i] = test_coeff[spot].copy()`
`A[i,j] = A[j,i]`

Reply:
```suggestion
                A[j,i] = test_coeff[spot].copy()
                A[i,j] = A[j,i]
```